src/Employee.java

```diff
@@ -23,11 +23,11 @@ public void setName(String name) {
   }
 
   public double getAverageMonthlySalary() {
-    return 0.0;
+    return calculatePay();
   }
 
-  public void calculatePay() {
-    getAverageMonthlySalary();
+  public double calculatePay() {
+    return 0.0;
```

Comment:
minor comment: one could make `Employee` class `abstract` to avoid any implementation of `calculatePay` method.